git-hub

```diff
@@ -1535,6 +1535,10 @@ class PullCmd (IssueCmd):
                 warnf('Checking out a closed pull request '
                     '(closed at {closed_at})!', **pull)
 
+            if pull['head']['repo'] is None:
+                die("Can't checkout pull request {}, no head repository found "
+                    "(maybe it was deleted?)", pull['number'])
```

Comment:
Yeah, it's OK, you could also do this `die("Can't checkout pull request {number}, no head repository found (maybe it was deleted?)", pull)`, but it's essentially the same.